src/deepness/dialogs/packages_installer/packages_installer_dialog.py

```diff
@@ -40,42 +41,49 @@ class PackageToInstall:
     import_name: str  # name while importing package
 
     def __str__(self):
-        return f'{self.name}=={self.version}'
+        return f'{self.name}{self.version}'
 
 
-# NOTE! For the time being requirement are repeated as in `requirements.txt` file
-# Consider merging it into some common file in the future
-# (if we can use a fixed version of pip packages for all python versions)
+REQUIREMENTS_PATH = 'https://raw.githubusercontent.com/PUTvision/qgis-plugin-deepness/master/requirements.txt'
```

Comment:
maybe we can ship `requirements.txt` along with the plugin?

Reply:
Otherwise old versions may stop working after we update the requirements someday, and it may be unexpected for the users